app/app.go

```diff
@@ -53,99 +43,42 @@ func New(options ...AppOption) *App {
 	return app
 }
 
-func (a *App) InitServer() {
-	a.srv.AppInitializedOnce.Do(func() {
-		a.initEnterprise()
```

Comment:
A bit lost here :sweat_smile: Where has `a.initEnterprise()` gone?

Reply:
It's all in `Server.initEnterprise()` now 😅